index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
+			const retryAfter = options !== undefined &&
+				['number', 'string'].includes(typeof options.retryAfter) ?
+				Number(options.retryAfter) :
+				2000;
+
+			setTimeout(() => {
```

Comment:
I think `const timeout = setTimeout(...); timeout.unref()` should be called:
  - if the process was already killed, we don't want to hold the process from exiting.
  - if it was not killed, the child process will hold the process from exiting, so no need for the timeout to do it as well.

Reply:
Ah, nice. Didn't know that existed. Added: b39acfdb29d37f064b448e54d50d5dad3610f6e8